File: Recursion/palindrome/palindrome.js
function checkPalindrome(str){
  if(str.length < 2) return 1;
    if(str.length === 2) if (str[0] === str[1]) return 1;
    if(str[0] === str.slice(-1)) return checkPalindrome(str.slice(1,-1));
    return 0;
}
class Solution {
    isPalindrome(str){
        //code here
      return checkPalindrome(str);
    }
}

File: Recursion/palindrome/palindrome.cpp
	int help(int l,int r,string &s){
	    if(l>=r) return 1;
	    if(s[l]!=s[r]) return 0;
	    return help(l+1,r-1,s);
	}
	int isPalindrome(string S)
	{
	    // Your code goes here
	    return help(0,S.size()-1,S);
	}
